gm_dotnet_native/dotnethelper-src/dotnethelper.cpp

```diff
@@ -45,7 +46,7 @@ std::string hosfxr_path = "garrysmod/lua/bin/dotnet/host/fxr/" + std::string(NET
 #ifdef WIN32
 void* hostfxr_library_handle = LoadLibraryA((hosfxr_path + "/hostfxr.dll").c_str());
 #elif __APPLE__
-void* hostfxr_library_handle = dlopen((hosfxr_path + "/libhostfxr.dylib").c_str(), RTLD_LAZY);
+void* hostfxr_library_handle = dlopen((hosfxr_path + "/libhostfxr.dylib").c_str(), RTLD_LAZY | RTLD_LOCAL | RTLD_LOCAL);
```

Comment:
Why `RTLD_LOCAL` is repeated?

Reply:
@SupinePandora43 It's indeed a typo. Fixed it.